### PR TITLE
Make Heroku deployment config clearer

### LINKED ISSUE #1: Adding an heroku account oauth redirects back to localhost:3000 instead of my configured url
callback url becomes: http://localhost:3000/ext/heroku/oauth/callback?code=

Adding through Account/Heroku/Add Account

If I change the url in the callback for the browser I get a set of private and public keys.

### LINKED ISSUE #677: Heroku auth redirecting to localhost:3000
After authenticating heroku iam being redirected to localhost. I've set environment variable SERVER_NAME to my app address still iam being redirected to localhost

http://localhost:3000/ext/heroku/oauth/callback?code=60e420f4-1414-436c-a025-25c5609067ef

Am i missing something?

### PR DESCRIPTION
**Pull request by terraboops**
fixes Strider-CD/strider#677
fixes Strider-CD/strider-heroku#1

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -12,3 +12,8 @@ To use on **anything other than localhost:3000**, you need to register your own
 This is a screenshot of what you will see on your dashboard:
 
 ![screenshot](docs/heroku_api_key.png)
+
+# Confifugring Strider to use your Heroku API Client
```
Comment (knownasilya) [inline Readme.md]: Typo :smile:

**Comment by terraboops** [comment]
epic pull request fail

**Comment by knownasilya** [comment]
@tylermauthe what happened?

**Comment by terraboops** [comment]
I pull requested into my own fork :trollface:

I've opened a proper pull request here: https://github.com/Strider-CD/strider-heroku/pull/7

**Pull request closed without merging**